.github/workflows/slash-commands.yml

```diff
@@ -27,6 +27,7 @@ jobs:
             build-connector
             publish-connector
             publish
```

Comment:
so is the `/publish` command still available after this PR? which workflow file does it use/how does it know to use it?

Reply:
This worfklow is responsible for writing the deprecation message: `.github/workflows/publish-command.yml`
Commands are mapped to workflow via the `-command` suffix in their filename.